deployment/sites-enabled/default.conf

```diff
@@ -20,6 +20,7 @@ server {
     # the domain name it will serve for
     server_name staging.projecta.kartoza.com changelog.linfiniti.com changelog.kartoza.com changelog.qgis.org changelog.inasafe.org staging.changelog.qgis.org;
     charset     utf-8;
+    include /home/web/media/server-name.txt
```

Comment:
I would imaging that since we will on a dedicated server using * for server name might be a simpler approach and then let projecta handle whether the hostname in the header of the request if valid or not.

Reply:
(that we we do not need to use server-name.txt tricks)